packages/core/components/ExternalInput/index.tsx

```diff
@@ -265,9 +265,14 @@ export const ExternalInput = ({
               </div>
             </div>
           </div>
-
-          <div className={getClassNameModal("footer")}>
-            {mappedData.length} result{mappedData.length === 1 ? "" : "s"}
+          <div className={getClassNameModal("footerContainer")}>
+            {field.renderFooter ? (
+              field.renderFooter(mappedData)
```

Comment:
Although this will work, it will stop React from being able to optimise correctly, creating a new component signature on each render and forcing a re-render.

I'd recommend moving this to the top of the component and wrapping it in a memo, then using JSX to render it.

Something like this (untested):

```tsx
const Footer = useMemo(() = (data) => (
  field.renderFooter ? (
    field.renderFooter(data)
  ) : (
    <span className={getClassNameModal("footer")}>
      {data.length} result{data === 1 ? "" : "s"}
    </span>
  )
), [field.renderFooter]);
```

```tsx
<Footer data={mappedData} />
```